arm-network/2016-09-01/swagger/network.json

```diff
@@ -1672,6 +1672,9 @@
             "description": "Parameters supplied to the create/update NetworkInterface operation"
           },
           {
+            "$ref": "#/parameters/LocationParameter"
```

Comment:
It would be great if you can let us know how are you consuming this spec? That will help us better serve you.